lib/tag-list/style.scss

```diff
@@ -108,138 +107,17 @@ button.tag-list-input {
 }
 
 .icon-reorder {
-  margin-right: 25px;
-}
-
-.editable-list {
-  display: flex;
-  flex-direction: column;
-  overflow: auto;
-  overflow-x: hidden;
-  -webkit-overflow-scrolling: touch;
-  margin: 0;
-  padding: 0;
-  list-style: none;
-
-  .editable-list-item {
-    display: flex;
-    flex: none;
-    padding: 0;
-    overflow: hidden;
-    white-space: nowrap;
-    -webkit-tap-highlight-color: transparent;
-    position: relative;
-    z-index: 10;
-    transition: $anim;
-
-    svg {
-      vertical-align: middle;
-    }
-  }
-
-  &.editable-list-reordering .editable-list-item {
-    transition: none;
-
-    * {
-      cursor: ns-resize;
-    }
-  }
-
-  .editable-list-item-reordering {
-    z-index: 0;
-  }
-
-  .editable-list-item-left {
-    display: flex;
-    align-items: center;
-    flex: 1 0 auto;
-    transition: $anim;
-    transform: translateX(0);
-  }
-
-  &.editable-list-removable .editable-list-item-left {
-    transform: translateX(-24px);
-  }
-
-  .editable-list-item-content {
-    display: flex;
-    flex: 1 1 auto;
-    align-items: center;
-  }
-
-  .editable-list-trash,
-  .editable-list-reorder {
-    cursor: pointer;
-    flex: none;
-    opacity: 0;
-    pointer-events: none;
-    transition: $anim;
-
-    svg {
-      transition: $anim;
-    }
-
-    &:focus {
-      outline: none;
-    }
-  }
-
-  .editable-list-trash {
-    width: 22px;
-    margin-right: 4px;
-
-    svg {
-      fill: $studio-red-40;
-    }
-
-    &:focus svg {
-      fill: darken($studio-red-40, 20%);
-    }
-  }
-
-  .editable-list-reorder {
-    display: flex;
-    align-items: center;
-    flex: none;
-    transform: translateX(24px);
-
-    svg {
-      fill: $studio-gray-50;
-    }
-
-    &:focus svg {
-      fill: $studio-gray-80;
-    }
-  }
-
-  &.editable-list-editing {
-    .editable-list-item-left {
-      transform: translateX(-4px);
-    }
-
-    .editable-list-trash {
-      opacity: 1;
-      pointer-events: auto;
-    }
-
-    .editable-list-reorder {
-      opacity: 1;
-      pointer-events: auto;
-      transform: translateX(0);
-    }
-  }
+  margin-right: 5px;
+  color: $studio-gray-10;
 }
 
 .theme-dark {
-  .tag-list .tag-list-input {
-    color: $studio-white;
+  .icon-reorder {
+    color: $studio-gray-50;
   }
-}
-
-.tag-list-item.theme-dark {
-  border-bottom: 1px solid $studio-gray-70;
-
-  svg.icon-reorder {
-    fill: $studio-gray-30;
+  .tag-list-title {
+    .panel-title {
+      color: $studio-white;
+    }
```

Comment:
Again, we can remove the title styles if we swap the PanelTitle component out for a plain H2.